Documentation/Debugger/gdb/Windows Subsystem for Linux.md

```diff
@@ -0,0 +1,88 @@
+# Windows 10's Windows Subsystem for Linux
+With the release of Windows 10 Creator's Update, you will now be able to use Visual Studio Code and the Microsoft C/C++ extension to debug your `Windows Subsystem for Linux (WSL)` Bash on Ubuntu projects.
```

Comment:
I think the official name is "Windows 10 Creators Update" (no apostrophe) - see https://blogs.windows.com/buildingapps/2017/04/05/windows-10-creators-update-creators-update-sdk-released/ for an example.